src/element.h

```diff
@@ -244,12 +246,22 @@ namespace litehtml
 
 	inline int litehtml::element::height() const
 	{
-		return m_pos.height + margin_top() + margin_bottom() + m_padding.height() + m_borders.height();
+		return margin_top() + margin_bottom() + content_height() + m_borders.height();
 	}
 
 	inline int litehtml::element::width() const
 	{
-		return m_pos.width + margin_left() + margin_right() + m_padding.width() + m_borders.width();
+		return margin_left() + margin_right() + content_width() + m_borders.width();
+	}
+
+	inline int litehtml::element::content_height() const

```

Comment:
The name is not right neither, the content area in the standard is the part covered by m_pos.

This length is the length of the border area